rest_typed_views/validators/validator_factory.py

```diff
@@ -81,8 +82,13 @@ def make_string_validator(cls, settings: ParamSettings):
             return serializers.IPAddressField(default=settings.default, protocol="both")
 
     @classmethod
-    def make_list_validator(cls, item_type: Any, settings: ParamSettings):
+    def make_list_validator(cls, item_type: Any, is_optional: bool, settings: ParamSettings):
         options = {"min_length": settings.min_length, "max_length": settings.max_length}
+        if is_optional:
+            default = settings.default
+            if settings.default == empty:
+                default = []
```

Comment:
@nitoqq Thanks, this looks great. One thing, though, do you think `None` makes more sense for the default if not specified?